native/src/utils/AppSettings.ts

```diff
@@ -16,6 +19,7 @@ export type SettingsType = {
   jpalTrackingEnabled: boolean | null
   jpalTrackingCode: string | null
   jpalSignals: Array<SignalType>
+  externalSourcePermissions: ExternalSourcePermission[]
```

Comment:
Maybe a key/value map would be easier to manage? No need to check for duplicates and so on.

Reply:
you can not use Maps in this template literal because `Map` is es6 @steffenkleinle